style/sidebar.css

```diff
@@ -0,0 +1,41 @@
+.Sidebar {
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 72px;
+  top: 55px;
+  font-size: 18px;
+  font-weight: bold;
+  color: brown;
+  background-color: white;
+
+  /* padding-left: 0; */
+  z-index: 200;
+  padding-left: 5px;
+}
+
```

Comment:
remove redednednt slectors

Reply:
- Good job suggesting this change 👏 .  Please note that it is important to sound friendly and polite while performing reviews. In this case, you could say "I'd suggest you remove redundant selectors" which sounds less than a command 😄 
- I'd advise that you give reasons for suggesting a change when performing reviews, this will help make the author understand why it is necessary to make that change. Kindly fix this 🙏

Reply:
okay